nexus/service/description.md

```diff
@@ -0,0 +1,8 @@
+Service Name:
```

Comment:
Is there any documentation we can link users to about this file format? I imagine a user seeing nexus for the first time might be confused on what they can do here.

Reply:
There's no public documentation yet. We plan to have markdown support in the UI and a template that will direct users what to put here.

Reply:
A template? Is this not freeform markdown that can have anything? Will there be an expectation of some format/values _inside_ the markdown?

Reply:
It is free form but we'd like to give some guidance on what would be useful to put in it.
This is likely a temporary step until we have code generation and a way to publish schemas to be explored in the UI.

Reply:
Makes sense. In my mind whatever is useful to them is useful to them and we don't need to mention what is useful to them, but I guess it doesn't matter much